src/server.js

```diff
@@ -10,10 +10,14 @@ const {createBlob} = require('./models/blob');
 const {createDigest, blobs, saveBlob} = require('./gameState');
 const {eventHandlers} = require('./events/eventHandlers');
 
+//my idea right here is to handle movement here on the digest. What we'll do is 
+//handle moving players blobs here in the timer. it could work, going to back up before trying
+
```

Comment:
This is fine for now, but I'd like to think about having some kind of dedicated manager for movement and maybe collision detection etc